app/templates/components/public/event-map.hbs

```diff
@@ -0,0 +1,11 @@
+<div class="ui grid fluid container">
```

Comment:
Please add these classes to the class definition at `event-map.js` and remove this div.

Ref: https://guides.emberjs.com/v2.13.0/components/customizing-a-components-element/